scene/gui/tree.cpp

```diff
@@ -4927,6 +4941,7 @@ void Tree::set_column_title_language(int p_column, const String &p_language) {
 	ERR_FAIL_INDEX(p_column, columns.size());
 	if (columns[p_column].language != p_language) {
 		columns.write[p_column].language = p_language;
+		//columns.write[p_column].cached_minimum_width_dirty = true;
```

Comment:
`title` and `language` (in some conditions, usually it's not) can change min. width, `text_direction` and `alignment` should not.

Reply:
`update_column()` invalidates the cache.

Reply:
I thought the point of adding these commented out lines here was to indicate that we might want to uncomment them in the future. But since `update_column` does indeed invalidate the cache already, I think they should be removed.